package.json

```
@@ -6,7 +6,7 @@
     "format": "pnpm biome format --write",
     "check": "pnpm biome check --write",
     "tsc": "astro sync && tsc --skipLibCheck",
-    "build": "astro build",
+    "build": "pnpm codegen all && astro build",
```

```suggestion
    "build": "astro build",
```

Not all the codegen is meant to be run on CI